corehq/apps/accounting/invoicing.py

```diff
@@ -83,9 +83,11 @@ def __init__(self, date_start, date_end, domain, recipients=None):
             raise InvoiceError("Domain '%s' is not a valid domain on HQ!" % domain)
 
     def create_invoices(self):
-        subscriptions = self._get_subscriptions()
-        self._ensure_full_coverage(subscriptions)
-        for subscription in subscriptions:
+        all_subscriptions = self._get_subscriptions()
+        self._ensure_full_coverage(all_subscriptions)
```

Comment:
future investigation if this is still needed? worth opening another ticket to clean this up

Reply:
https://dimagi.atlassian.net/browse/SAAS-16270